src/components/resource/AdminMenu.jsx

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+import React from 'react/addons';
+import {DropdownButton, MenuItem} from 'react-bootstrap';
+import classNames from 'classnames';
+
+import 'font-awesome/css/font-awesome.css';
```

Comment:
Since we don't use webpack for styles in our apps, this causes font awesome to be included twice. So the convention right now is to let the app include the css, and assume it is present.